components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+/**
+ An implementation of Material Tabs (https://material.io/design/components/tabs.html).
+ */
+@interface MDCTabBarView : UIScrollView
```

Comment:
Is it possible to not subclass UIScrollView? The alternative being to own and manage a scroll view internally.

Reply:
Why is that preferred? It sounds more complex than the current structure.

Reply:
I can't seem to win in a conversation with @codeman7 so I will switch this declaration.  I haven't determined how the component will work with a UIView and embedded scroll view so it's increasing the risk of the project. However, since I don't have time to debate this all day long I'm going to do whatever it takes to move forward.

Reply:
Done.